pyrokinetics/templates/input.gs2

```diff
@@ -75,8 +78,8 @@
     mass = 2.724e-4
     dens = 1.0
     temp = 1.0
-    tprim = 3.0
-    fprim = 1.0
+    tprim = 9.0
```

Comment:
We need to be careful here and pyro doesn't quite deal with using `Rmaj` as the reference length yet. It should be doable given our normalisation setup.

Reply:
So in these templates, does it matter what the normalizing length-scale is?   I was thinking that the templates should work as is if we ran them, so while I can re-normalize everything to `a_minor`, `GKW` won't like it (`Rmaj` normalization is hardwired)...

Reply:
Sure but `TGLF` and `CGYRO` (and `GFTM` when that gets added in) have `a_minor` hardwired as the normalising length case. Given that more codes have `a_minor` I'm inclined to use that in the templates. I do accept that the `GKW` should be physically correct so to have `Rmaj`, such that if we ran we it would be physically the same as the others. 

This is a slightly fiddily I admit

Reply:
1. Should we keep `GKW` in `Rmaj` and `TGLF` & `CYGRO` in `a_minor` then?  Should be fine as long as the templates can produce the same result?
2. Following on from @ZedThree's suggestion, do you want the templates to be linear for now?  A more comprehensive template (perhaps covering nonlinear settings) could be a separate PR?